File: app/src/main/java/com/diploma/bank/MainMenu/MainActivity.java
package com.diploma.bank.MainMenu;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.design.widget.BottomNavigationView;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.view.MenuItem;

import com.diploma.bank.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        BottomNavigationView navigation = (BottomNavigationView) findViewById(R.id.bottom_navigation);
        navigation.setOnNavigationItemSelectedListener(mOnNavigationItemSelectedListener);

        loadFragment(MainHomeFragment.newInstance());
    }

    //Нижняя панель навигации
    private BottomNavigationView.OnNavigationItemSelectedListener mOnNavigationItemSelectedListener
            = new BottomNavigationView.OnNavigationItemSelectedListener() {
        @Override
        public boolean onNavigationItemSelected(@NonNull MenuItem item) {
            switch (item.getItemId()) {
                case R.id.action_home:
                    loadFragment(MainHomeFragment.newInstance());
                    return true;
                case R.id.action_payment:
                    loadFragment(MainPaymentFragment.newInstance());
                    return true;
                case R.id.action_history:
                    loadFragment(MainHistoryFragment.newInstance());
                    return true;
                case R.id.action_more:
                    loadFragment(MainAdditionalFragment.newInstance());
                    return true;
            }
            return false;
        }
    };

    //Загрузка фрагмента
    private void loadFragment(Fragment fragment) {
        FragmentTransaction ft = getSupportFragmentManager().beginTransaction();
        ft.replace(R.id.fl_content, fragment);
        ft.commit();
    }
}

File: app/src/main/java/com/vmbeshenov/bank/MainMenu/MainActivity.kt
package com.vmbeshenov.bank.MainMenu

import android.os.Bundle
import android.support.design.widget.BottomNavigationView
import android.support.v4.app.Fragment
import android.support.v7.app.AppCompatActivity
import android.view.View
import com.vmbeshenov.bank.R

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val navigation = findViewById<View>(R.id.bottom_navigation) as BottomNavigationView
        navigation.setOnNavigationItemSelectedListener(mOnNavigationItemSelectedListener)
        loadFragment(MainHomeFragment.newInstance())
    }

    //Нижняя панель навигации
    private val mOnNavigationItemSelectedListener = BottomNavigationView.OnNavigationItemSelectedListener { item ->
        when (item.itemId) {
            R.id.action_home -> {
                loadFragment(MainHomeFragment.newInstance())
                return@OnNavigationItemSelectedListener true
            }
            R.id.action_payment -> {
                loadFragment(MainPaymentFragment.newInstance())
                return@OnNavigationItemSelectedListener true
            }
            R.id.action_history -> {
                loadFragment(MainHistoryFragment.newInstance())
                return@OnNavigationItemSelectedListener true
            }
            R.id.action_more -> {
                loadFragment(MainAdditionalFragment.newInstance())
                return@OnNavigationItemSelectedListener true
            }
        }
        false
    }

    //Загрузка фрагмента
    private fun loadFragment(fragment: Fragment) {
        val ft = supportFragmentManager.beginTransaction()
        ft.replace(R.id.fl_content, fragment)
        ft.commit()
    }
}